target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+    compatible = "zte,mf286c3", "mediatek,mt7621-soc";
+    model = "ZTE MF286C3";
+
```

Comment:
Please fix indents and use tabs instead of the whitespaces. And use unix stile for the line endings.